.travis.yml

```diff
@@ -1,47 +1,28 @@
 language: ruby
 rvm:
-  - '1.9.3'
-  - '2.0.0'
-  - '2.1.1'
-  - '2.2.7'
-  - '2.3.4'
+  - '2.4'
+  - '2.5'
+  - '2.6'
+  - '2.7'
   - ruby-head
 services:
   - redis-server
 gemfile:
-  - Gemfile.rails-4.0.x
-  - Gemfile.rails-4.1.x
-  - Gemfile.rails-4.2.x
-  - Gemfile.rails-5.0.x
-  - Gemfile.rails-5.1.x
+  - Gemfile.rails-5.2.x
+  - Gemfile.rails-6.0.x
   - Gemfile.rails-head
 matrix:
   allow_failures:
     - rvm: '1.9.3'
     - rvm: ruby-head
     - gemfile: Gemfile.rails-head
   exclude:
-    - rvm: '1.9.3'
-      gemfile: Gemfile.rails-5.0.x
-    - rvm: '1.9.3'
-      gemfile: Gemfile.rails-5.1.x
-    - rvm: '2.0.0'
-      gemfile: Gemfile.rails-5.0.x
-    - rvm: '2.0.0'
-      gemfile: Gemfile.rails-5.1.x
-    - rvm: '2.1.1'
-      gemfile: Gemfile.rails-5.0.x
-    - rvm: '2.1.1'
-      gemfile: Gemfile.rails-5.1.x
-    - rvm: '2.2.7'
-      gemfile: Gemfile.rails-4.0.x
-    - rvm: '2.2.7'
-      gemfile: Gemfile.rails-4.1.x
-    - rvm: '2.2.7'
-      gemfile: Gemfile.rails-4.2.x
-    - rvm: '2.3.4'
-      gemfile: Gemfile.rails-4.0.x
-    - rvm: '2.3.4'
-      gemfile: Gemfile.rails-4.1.x
-    - rvm: '2.3.4'
-      gemfile: Gemfile.rails-4.2.x
\ No newline at end of file
+    - rvm: 2.4
+      gemfile: Gemfile.rails-6.0.x
+    - rvm: 2.4
+      gemfile: Gemfile.rails-head
+
+
+
+
+      
```

Comment:
Den überflüssigen Whitespace hier gerne entfernen. Wir könnten überlegen, hier EditorConfig und Linting einzubauen, um in Zukunft nicht weiter über Formatierungsfragen reden zu müssen.